packages/contracts/deploy/helpers.ts

```diff
@@ -2,6 +2,7 @@ import {promises as fs} from 'fs';
 import {ethers} from 'hardhat';
 import {BigNumberish} from 'ethers';
 import {HardhatRuntimeEnvironment} from 'hardhat/types';
+import IPFS from 'ipfs-http-client';
```

Comment:
FYI: We could also use our own IPFS cluster client `@aragon/sdk-ipfs`

Reply:
yes, tested that,
but the `@aragon/sdk-ipfs` is not designed to be used with node
and it require installing multiple libraries beside it, to make it work,
i am not sure if it worth for this task.